.github/workflows/deploy.yml

```diff
@@ -23,8 +23,7 @@ env:
 
   # Note: breaking up tests so that no test cases hang on Windows, to be reverted
   CIBW_TEST_COMMAND: |
-    pl-device-test --device=lightning.qubit --skip-ops -x --tb=short --no-flaky-report -k "not test_four_qubit_random_circuit"
-    pl-device-test --device=lightning.qubit --skip-ops -x --tb=short --no-flaky-report -k "test_four_qubit_random_circuit"
+    pl-device-test --device=lightning.qubit --skip-ops -x --tb=short --no-flaky-report

```

Comment:
Oh I didn't realize this, it was hanging for the `test_four_qubit_random_circuit` test? :thinking:

Reply:
Yep! It was hanging after having executed 4 or 5 test cases on `test_four_qubit_random_circuit` 🤔